public/create_meeting.php

```diff
@@ -15,7 +15,6 @@
     $meeting->planned_time = $_POST["time"];
     $meeting->has_passed = '0';
     if (isset($_POST["team"])) {
-        echo "here";
         echo $_POST["team"];
```

Comment:
noooo you only removed one of the echos 😂

Reply:
omg 🤦